### DISCUSSION TITLE
xiaomiKEY和一些秒解BL的是什么原理啊，谁能讲讲吗

### DISCUSSION THREAD

**Discussion by wangchuanyes** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#40] [Q&A] [answered]
秒解小米14 Bootloader锁视频https://www.youtube.com/watch?v=MUsQhmWq_kE
xiaomiKEY官网https://xiaomikey.net

**Comment by MlgmXyysd** [comment]
有内鬼呗

**Comment by mrzzoxo** [comment]
米家售后工号

**Comment by ghost** [comment]
这玩意服务器不在线

**Comment by coldshineb** [comment, marked as answer]
海外小米售后账号搭建的自动化解锁服务器，手机进bootloader后获取一次性的设备token，解锁时token发至服务器拿到解锁码文件，本地进行解锁

**Reply by xbn2002** [reply]
据说是印度工厂，群内消息不一定可靠

**Reply by yufeng5461** [reply]
有没有大佬能写一个类似的脚本啊

**Reply by 17Extreme** [reply]
这如果能写个脚本出来的话，也不至于去答那种贼难的题

**Reply by xbn2002** [reply]
> 这如果能写个脚本出来的话，也不至于去答那种贼难的题

想多了，手机这边解锁需要小米服务器用他的私钥对你的设备token进行签名。没有私钥怎么签名？除非有人有本事破解RSA算法
![AF9CF8AB6AC99B35118C7441E27B77B4](https://github.com/user-attachments/assets/591bf678-7d22-4e49-bdbf-c89285aeb197)

**Reply by 17Extreme** [reply]
听着好复杂

**Reply by duncai233** [reply]
小米是怎么实现那个单次性的

**Reply by bingqiu456** [reply]
> 小米是怎么实现那个单次性的

进bootloader的时候生成一些随机内容 然后把随机内容发给服务器 服务器返回key 然后给bootloader校验

**Comment by baizhi958216** [comment]
魅族教的:)
